ui-tests/src/test/java/org/openmrs/reference/page/EditReportPage.java

```diff
@@ -0,0 +1,42 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class EditReportPage extends Page {
+	
+	private static final By PREVIEW_BUTTON = By.cssSelector("#previewButton");
+	private static final By START_DATE_FIELD = By.cssSelector("#startDate");
+	private static final By END_DATE_FIELD = By.cssSelector("#endDate");
+	private static final By RUN_BUTTON = By.cssSelector("#preview-parameterizable-button");
+	private static final By CLOSE_TAB = By.xpath("//span[contains(text(),'close')]");
+	
+	public EditReportPage(Page parent) {
+	    super(parent);
+	}
+	
+	public void clickOnPreviewButton() {
+	    clickOn(PREVIEW_BUTTON);
+	}
+	
+	public void enterStartDate(String startDate) {
+	    setTextToFieldNoEnter(START_DATE_FIELD, startDate);
+	}
+	
+	public void enterEndDate(String endDate) {
```

Comment:
Here are the references incase [here](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/page/VisitNotePage.java#L46) and [here](https://github.com/openmrs/openmrs-contrib-qaframework/blob/91cc83e903059428559db923e7dd938f8db31076/src/test/java/org/openmrs/contrib/qaframework/automation/ClinicalVisitSteps.java#L92)